### DISCUSSION TITLE
luat后台任务

### DISCUSSION THREAD

**Discussion by jokokiswan** [openLuat/LuatOS#80] [Q&A] [answered]
是否可以在 luat 中使用计时器启动后台任务？

**Comment by chenxuuu** [comment, marked as answer]
没看明白，详细说说？

**Reply by jokokiswan** [reply]
在stm32中，我们可以在不干扰其他进程的情况下对传感器进行采样，这是否也适用于luatos？

**Reply by chenxuuu** [reply]
luatos的task框架支持协程实现多任务：https://wiki.luatos.com/luaGuide/luatask.html

**Reply by jokokiswan** [reply]
谢谢
